internal/querycoord/task.go

```diff
@@ -2175,6 +2175,10 @@ func (lbt *loadBalanceTask) execute(ctx context.Context) error {
 						Schema:       collectionInfo.Schema,
 						CollectionID: collectionID,
 						ReplicaID:    lbt.replicaID,
+						LoadMeta: &querypb.LoadMetaInfo{
+							CollectionID: collectionID,
+							PartitionIDs: []int64{partitionID},
```

Comment:
`PartitionIDs` shall be the full list of the loaded partitions

Reply:
> `PartitionIDs` shall be the full list of the loaded partitions

what's the full list of a loadSegmentRequest?